src/component/page/bruker-detaljer/deltaker-detaljer/DeltakelseInfo.tsx

```diff
@@ -41,12 +42,24 @@ export const DeltakelseInfo = ({
 		skjulDeltakerPromise.setPromise(skjulDeltaker(deltaker.id))
 	}
 
-	const skalViseDeltakelsesprosent = [ Tiltakskode.ARBFORB, Tiltakskode.VASV ]
+	const skalViseDeltakelsesmengde = [ Tiltakskode.ARBFORB, Tiltakskode.VASV ]
 		.includes(deltaker.tiltakskode)
 
 	const kanFjerneDeltaker = [ TiltakDeltakerStatus.IKKE_AKTUELL, TiltakDeltakerStatus.HAR_SLUTTET, TiltakDeltakerStatus.AVBRUTT, TiltakDeltakerStatus.FULLFORT ]
 		.includes(deltaker.status.type)
 
+	const getDeltakelsesmengdetekst = (deltakelseProsent: Nullable<number>, dagerPerUke: Nullable<number>): string => {
+		if ((deltakelseProsent === null || deltakelseProsent === undefined) && (dagerPerUke === null || dagerPerUke === undefined || dagerPerUke < 1 || dagerPerUke > 5)){
+			return 'Ikke satt'
+		} else if (deltakelseProsent === 100 || dagerPerUke === null || dagerPerUke === undefined || dagerPerUke < 1 || dagerPerUke > 5) {
```

Comment:
her kan du sjekke `!dagerPerUke` isteden for å sjekke både null og undefined, stusser litt på at den kan være begge deler da men nå her jeg ikke sett over hele pr'en enda

Reply:
Jeg tror den kan være både null og undefined siden jeg bruker Nullable..? Men godt tips dette med !dagerPerUke, det skal jeg endre! 🙌